File: c atau c++/pembelian.cpp
#include <iostream>
using namespace std;

main() {
    string barang;
    float harga;
    int jumlah;

    cout << "Masukkan Nama Barang : ";
    cin >> barang;
    cout << "Masukkan Harga Barang : ";
    cin >> harga;
    cout << "Masukkan Jumlah Barang : ";
    cin >> jumlah;

    int total = harga * jumlah;

    int diskon = total * 0.02;
    if (total<=19200) {
        diskon = 0;
    }

    int total_pembayaran = total - diskon;

    cout << "========== STRUK PEMBELIAN ==========" << endl;
    cout << "Nama Barang : " << barang << endl;
    cout << "Harga Satuan : " << harga << endl;
    cout << "Jumlah Beli : " << jumlah << endl;
    cout << "Total Harga : " << total << endl;
    cout << "Diskon : " << diskon << endl;
    cout << "Total Pembayaran : " << total_pembayaran << endl;
    cout << "=====================================" << endl;

    return 0;

}

File: php/pembelian.php
<?php

$barang = readline("Masukkan Nama Barang: ");
$harga = floatval(readline("Masukkan Harga Barang: "));
$jumlah = intval(readline("Masukkan Jumlah Barang: "));

$total = $harga * $jumlah;

if ($total > 19200) {
    $diskon = $total * 0.02;
} else {
    $diskon = 0;
}

$total_pembayaran = $total - $diskon;

echo "\n===== STRUK PEMBELIAN =====\n";
echo "Nama Barang   : $barang\n";
echo "Harga Satuan  : $harga\n";
echo "Jumlah Beli   : $jumlah\n";
echo "Total Harga   : $total\n";
echo "Diskon        : $diskon\n";
echo "Total Pembayaran   : $total_pembayaran\n";
echo "============================\n";
?>